velox/exec/tests/SpillerTest.cpp

```diff
@@ -175,30 +185,58 @@ class SpillerTest : public exec::test::RowContainerTestBase {
     return data;
   }
 
-  folly::IOThreadPoolExecutor* FOLLY_NONNULL executor() {
+  folly::IOThreadPoolExecutor* executor() {
     static std::mutex mutex;
     std::lock_guard<std::mutex> l(mutex);
-    if (!executor_) {
-      executor_ = std::make_unique<folly::IOThreadPoolExecutor>(8);
+    if (GetParam() == 0) {
+      return nullptr;
+    }
+    if (executor_ == nullptr) {
+      executor_ = std::make_unique<folly::IOThreadPoolExecutor>(GetParam());
     }
     return executor_.get();
   }
 
+  const int numPartitions_ = 4;
   std::unique_ptr<folly::IOThreadPoolExecutor> executor_;
 };
 
-TEST_F(SpillerTest, spilFew) {
+TEST_P(SpillerTest, spilFew) {
   testSpill(10);
 }
 
-TEST_F(SpillerTest, spilMost) {
+TEST_P(SpillerTest, spilMost) {
   testSpill(60);
 }
 
-TEST_F(SpillerTest, spillAll) {
+TEST_P(SpillerTest, spillAll) {
   testSpill(100);
 }
 
-TEST_F(SpillerTest, error) {
+TEST_P(SpillerTest, error) {
   testSpill(100, true);
 }
+g
+#ifndef NDEBUG
```

Comment:
Why do we need this check?

Reply:
The reason is that we run unit test in release build in some setup in CI. And test value doesn't work in release build but only for debug build.